src/lightning/pytorch/demos/lightning_transformer.py

```diff
@@ -0,0 +1,29 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+import lightning.pytorch as pl
+from lightning.pytorch.demos.transformer import Transformer
```

Comment:
I would define this LightningModule together with the nn.Module Transformer

Reply:
moved in [05459ca](https://github.com/Lightning-AI/lightning/pull/18452/commits/05459caf327ed834c88f61e1dbefcc4a0a4b240f)